ARK Server Manager/ServerSettingsControl.xaml

```diff
@@ -905,17 +905,22 @@
                             <GroupBox Grid.Row="10" Grid.ColumnSpan="6" Header="Server Options">
                                 <Grid>
                                     <Grid.ColumnDefinitions>
-                                        <ColumnDefinition/>
-                                        <ColumnDefinition/>
+                                        <ColumnDefinition Width="67*"/>
```

Comment:
Did you mean to use the * notation here, which sets relative widths, or do you intend absolute widths?